Microsoft.Toolkit.Uwp.SampleApp/SamplePages/InfiniteCanvas/InfiniteCanvasPage.xaml.cs

```diff
@@ -81,6 +83,60 @@ private void Load()
                     }
                 }
             });
+
+            SampleController.Current.RegisterNewCommand("Export max view as image", async (sender, args) =>
+            {
+                if (_infiniteCanvas != null)
+                {
+                    var savePicker = new Windows.Storage.Pickers.FileSavePicker
```

Comment:
Can you using Windows.Storage.Pickers?

Reply:
@lindexi to clarify, you mean add a using at the top and not have the full namespace here?